packages/protocol/contracts/common/Accounts.sol

```diff
@@ -148,7 +157,10 @@ contract Accounts is
    *      is 0x0 or msg.sender).
    */
   function setWalletAddress(address walletAddress, uint8 v, bytes32 r, bytes32 s) public {
-    require(isAccount(msg.sender), "Unknown account");
+    require(
+      isAccount(msg.sender),
+      string(abi.encodePacked(msg.sender, " is not a registered account"))
```

Comment:
It's not as useful as it seems to include the `msg.sender` in the revert reason
In order to get the revert reason, clients use `eth_call` RPC which takes as input the encoded transaction
This encoded transaction already contains enough information to derive the `sender`

Reply:
cc @nambrot @deepakhb2
Any gas overhead isn't worth it imo

Reply:
I don't think we care about consuming ~1k extra gas when the tx reverts, but would prefer not to consume that gas if the tx succeeds. Does the string concatenation actually cost any gas if the condition is true? I suspect it compiles the same as:
```
if (!isAccount(msg.sender)) {
  revert(string(abi.encodePacked(msg.sender, " is not a registered account")))
}
```

Reply:
my recommendation is to make the revert reason static and avoid the question altogether
don't see much downside
```suggestion
      "sender is not a registered account"
```

Reply:
That was @m-chrzan 's theory as well

Reply:
@yorhodes Initially, I thought of it but I didn't raise as I wasn't sure about actual use case of this message. I think, sender will be know the address used for the transaction and It makes sense to respond with 'Sender is not a registered account'. I will make this change as well if this is the final call.